### ISSUE TITLE
dialog_position node isn't documented for say command

### ISSUE THREAD

**Issue by balloonpopper** [godot-escoria/escoria-issues#200]
**Describe the bug**
Discovered this code in floating.gd
		rect_position = _current_character.get_node(
			"dialog_position"

What effect a "dialog_position" node is / how to use it is not documented

**Comment by dploeger** [comment]
This is already documented in the simple dialog plugin: https://github.com/godot-escoria/escoria-dialog-simple/tree/develop

However, we might add the docs of the stock plugins to the official documentation somewhere or at least link to them.

**Comment by balloonpopper** [comment]
Right, I hadn't noticed that was a part of an addon. I think linking the stock plugins to the official documentation (and having links in individual commands like "say" that are extended by the stock plugins) would be helpful.
